packages/monaco/package.json

```diff
@@ -10,6 +10,7 @@
     "@theia/markers": "^0.3.19",
     "@theia/outline-view": "^0.3.19",
     "@theia/workspace": "^0.3.19",
+    "deepmerge": "2.0.1",
```

Comment:
@marcdumais-work we already have deepmerge in yarn.lock via wdio dev dependency, do i need to open another CQ?

Reply:
Hi @akosyakov,

Good news: no need for a new or updated CQ: "deepmerge": "2.0.1" was already covered in recent [CQ 18717](https://dev.eclipse.org/ipzilla/show_bug.cgi?id=18717)

Reply:
We should probably have waited until the CQ above was approved or given "merge permission" before merging this PR. In theory if this new dependency is not license-compatible, we would need to take action to remove it from the repo... (should be ok in this specific case though)

Reply:
Next time I'll add a review comment on the PR to reflect this situation.